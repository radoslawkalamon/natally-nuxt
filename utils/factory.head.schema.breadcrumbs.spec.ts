import { factoryHeadSchemaBreadcrumbs } from '@/utils/factory.head.schema.breadcrumbs'

describe('Utils / Factory / Head / Schema / Breadcrumbs', () => {
  test('shall match snapshot', () => {
    const args = {
      breadcrumbs: [
        {
          name: 'Test',
          path: '/test/'
        },
        {
          name: 'Path',
          path: '/test/path/'
        }
      ],
      path: '/test/path/'
    }
    expect(factoryHeadSchemaBreadcrumbs(args)).toMatchSnapshot()
  })
})
