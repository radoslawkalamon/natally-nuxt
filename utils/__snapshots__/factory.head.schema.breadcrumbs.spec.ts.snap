// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utils / Factory / Head / Schema / Breadcrumbs shall match snapshot 1`] = `
{
  "@id": "https://169cm.pl/test/path/#breadcrumb",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "item": "https://169cm.pl/",
      "name": "169cm.pl",
      "position": 1,
    },
    {
      "@type": "ListItem",
      "item": "https://169cm.pl/test/",
      "name": "Test",
      "position": 2,
    },
    {
      "@type": "ListItem",
      "item": "https://169cm.pl/test/path/",
      "name": "Path",
      "position": 3,
    },
  ],
}
`;
